L1Trigger/L1THGCal/python/hgcalTowerMapProducer_cfi.py

```diff
@@ -22,10 +22,10 @@
 
 hgcalTowerMapProducer = cms.EDProducer(
     "HGCalTowerMapProducer",
-    InputTriggerCells = cms.InputTag('hgcalVFEProducer:HGCalVFEProcessorSums'),
+    InputTriggerSums = cms.InputTag('hgcalConcentratorProducer:HGCalConcentratorProcessorSelection'),
     ProcessorParameters = tower_map.clone()
     )
 
 hgcalTowerMapProducerHFNose = hgcalTowerMapProducer.clone(
-    InputTriggerCells = cms.InputTag('hfnoseVFEProducer:HFNoseVFEProcessorSums')
+    InputTriggerSums = cms.InputTag('hfnoseConcentratorProducer:HGCalConcentratorProcessorSelection')
```

Comment:
The producer name shouid be `hgcalConcentratorProducerHFNose` instead of `hfnoseConcentratorProducer`
https://github.com/PFCal-dev/cmssw/blob/0b364540b912273432880010291aa011e00f5933/L1Trigger/L1THGCal/python/hgcalConcentratorProducer_cfi.py#L166-L169
(the naming is not super consistent I agree)